api/Dockerfile

```diff
@@ -1,33 +1,29 @@
 # Remember: the context for this build is the root of the monorepo,
 # not the api folder
 
-# TODO: we only use bun as a dependency manager, but api is still executed with
-# node. We should eventually get rid of node
-FROM oven/bun:slim as base
+# We need Node to run prisma generate. See https://github.com/oven-sh/bun/issues/3827
+# TODO: switch to oven/bun-node:slim
+FROM imbios/bun-node:latest-current-slim as base
```

Comment:
@neilsong: I purposefully used `oven/bun` directly so there's minimal underlying magic and future migration less break-prone. Does nvm not work?

Reply:
Yea for some reason it’s hella cooked for me. Prisma is being cringe as usual.

Future migration should be a simple change from this image to bun. This image only just has node and bun side-by-side.

Reply:
If you want, we can take a look at that image and copy over their code for installing node with bun but I feel like that’s kinda unnecessarily reinventing the wheel ykwim?

Reply:
I know, I started by using this image. But:

1. No control of Node/Bun version and no guarantee that it will be latest and/or customizable
2. It's 3rd party

Reply:
Alri - I mean it uses a GH action to automatically tag and push versioned images based on bun updates but it could break if bun changes their update process.

I’ll take a look at their Dockerfile and see what I can do

Reply:
Their dockerfile installs bun on top of node.

Since we only need the node binary, let’s just copy it over from the node docker image into the base image - what do you think?

Reply:
Erm—I still felt like using nvm is the most controllable and robust way, but otherwise using this image is probably better.

Reply:
kk - i'll work out nvm

Reply:
ok this thing is getting way too complicated because of the way docker manages shells and sessions. nvm relies on session-specific env vars that get wiped in weird ways cuz docker. i think we just go with this image.